packages/react-dom-bindings/src/client/ReactDOMInput.js

```diff
@@ -131,7 +131,6 @@ export function updateInput(
     // Submit/reset inputs need the attribute removed completely to avoid
     // blank-text buttons.
     node.removeAttribute('value');
-    return;
```

Comment:
I noticed that this return was problematic because we don't restore the `name` after setting it to empty.